content/accessibility/index.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: Accessibility
+---
+
+import {Grid, Flex, Box, Link, Text, Label} from '@primer/components'
+
+
+<Grid gridTemplateColumns={['1fr', null, null, null, '1fr']} gridGap={4}>
+    
+    <div>
+        <Link href="/design/content/accessibility/accessibility-at-github.mdx" fontWeight="bold">Accessibility at GitHub</Link>
+        <Box as="p">GitHub's aim is to be the home for all developers. To be inclusive means we must consider accessibility at the core of how we design. GitHub aims for Web Content Accessibility Guidelines (WCAG) 2.1 AA compliance. This includes all of WCAG 2.0 AA plus additional considerations.</Box>
```

Comment:
This should be a very short summary of what the page includes, instead of stating guidelines.

Reply:
How about this? 

```suggestion
        <Box as="p">Get started with accessibility at GitHub</Box>
```